src/web/TheController.java

```diff
@@ -78,10 +78,14 @@ public void viewPost(@RequestParam(value="id") int id,
 		//TODO: load post here
 		
 		int commentCtr = CommentManager.commentCount(id);
+		Post post = PostManager.getPost(id);
 		
 		//TEMPORARY
```

Comment:
you can do request.setAttribute("post",post);

then in the JSP page you can do ${post.id},${post.title},${post.author},${post.content}, and ${post.date}